src/Symfony/Component/HttpKernel/EventListener/RouterListener.php

```diff
@@ -140,7 +140,11 @@ public function onKernelRequest(GetResponseEvent $event)
             }
 
             if (null !== $this->logger) {
-                $this->logger->info(sprintf('Matched route "%s"', $parameters['_route']), $parameters);
+                $logContext = array(
+                    'route-params' => $parameters,
+                    'request-uri' => $request->getUri(),
+                );
```

Comment:
#13418 tries to make context more consistent. So, can you rename `route-params` to `route_parameters` and `request-uri` to `request_uri`? You can also inline the context instead of creating a temp var here. Thanks.